scripts/pre-push

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+// requirements
+const ethereumjs = require('ethereumjs-util')
+const https = require('https')
+const exec = require('child_process').exec
+// constants
+const host = 'https://api.etherscan.io/api?module=account&action=balancemulti&address='
+// variables
+var foundPrivateKeys = []
+var formatedKeys = []
+var rawKeysFound = []
+var keyFound = false
+// start script
+console.log('\x1b[1m', 'Checking for possible private keys with balance...\n', '\x1b[0m')
+// setup shell script command function
+var result = function (command, cb) {
+  exec(command, function (err, stdout, stderr) {
+    if (err != null) {
+      return cb(new Error(err), null)
+    } else if (typeof (stderr) !== 'string') {
+      return cb(new Error(stderr), null)
+    } else {
+      return cb(null, stdout)
+    }
+  })
+}
+// get git to return files that have been altered
+result('git diff --name-only', function (err, response) {

```

Comment:
I don't see the argument `${GIT_PARAMS}` being used in this script. How will git diff know what to compare against?